packages/components/src/types/common.ts

```diff
@@ -0,0 +1,13 @@
+export type TextWithA11y = {
+  /** Text to display */
+  text: string
+  /** Accessibility override for text */
+  a11y?: string
```

Comment:
This should likely be named `a11yLabel` or `label` or something to facilitate inclusion of other a11y overrides that can apply to text (hint, value, etc.) later.

Reply:
Renamed `a11yLabel`